toolbox/CMakeLists.txt

```diff
@@ -244,6 +253,10 @@ else()
     target_link_libraries(WBToolbox ${LINKED_LIBRARIES} ${Matlab_LIBRARIES})
 endif()
 
+# Remote Inverse Kinematics requires C++11
+set(CXX_STANDARD_REQUIRED ON)
+set_property(TARGET WBToolbox PROPERTY CXX_STANDARD 11)
```

Comment:
I suggest to use this:
~~~
if (CMAKE_VERSION VERSION_LESS "3.1")
    if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR 
        CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
        target_compile_options(WBToolbox "--std=c++11")
    endif()
else()
    set(CXX_STANDARD_REQUIRED ON)
    set_property(TARGET WBToolbox PROPERTY CXX_STANDARD 11)
endif()
~~~
to preserve compatibility with Ubuntu 14.04 that has CMake 2.8.12 .